ppl/archive/index/defstore.go

```diff
@@ -0,0 +1,142 @@
+package index
+
+import (
+	"context"
+	"sync"
+
+	"github.com/brimsec/zq/pkg/iosrc"
+	"github.com/brimsec/zq/zqe"
+	"github.com/segmentio/ksuid"
+)
+
+const defsDir = "indexdefs"
+
+type DefStore struct {
+	defs Defs
+	dir  iosrc.URI
+	lut  DefMap
+	mu   sync.Mutex
```

Comment:
I don't think this structure should need/have a mutex, as the source of information will always be reading the contents of the indexdefs directory.

Reply:
Dropped the DefStore store.